src/main/java/org/example/homework_nr_5/Weekday.java

```diff
@@ -0,0 +1,38 @@
+package org.example.homework_nr_5;
+
+public enum Weekday {
+
+    MONDAY,
+    TUESDAY,
+    WEDNESDAY,
+    THURSDAY,
+    FRIDAY,
+    SATURDAY,
+    SUNDAY;
+
+        public boolean isWeekday() {
+            for (int index = 0; index != Weekday.values().length; index++)
```

Comment:
No need to loop, use `this` keyword instead
Also, no need for second return

Reply:
It seems you've confused pull request. We are talking about homework No 5 now (but the pull request is for homework6).
I did 2 commits in homework 5 initially - 1st version was done  using "this" and another one was with loops. I amended it again anyway.